ChrisConquet/GhostMove.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class GhostMove : MonoBehaviour {
+
+
+    [SerializeField]
+    Transform _destination;
+    private NavMeshAgent agent;
+    private int lifetime = 2;
+
+    private float accuracyWP = 0.6f; // accuracy when following way points
+
+    public enum State { Dead, Aggro, Patrol };
+    public State enumState;
+
+
+
+    private float speed = 15;
+
+    public GameObject[] waypoints;
+    public int currentWP = 0;
+
+
+    public Transform player;
+    Vector3 direction;
+
+
+
+
+    // Use this for initialization
+    void Start() {
+

```

Comment:
Ik vindt al die extra Enters een beetje onnodig.
(Misschien verward met de code conventions?)
Maar dat is mijn eigen stijl.
Verder ziet alles er goed uit.